packages/telescope/__tests__/misc.test.ts

```diff
@@ -48,12 +48,14 @@ const options: TelescopeOptions = {
     typingsFormat: {
       customTypes: {
         useCosmosSDKDec: true,
+        // default uses cosmjs/math Decimals, undefined uses agoric-sdk
+        useAgoricDecimal: 'default',
```

Comment:
add useAgoricDecimal option to misc test file.

Reply:
the comment is removed please ignore.

to clarify default uses agoric-sdk, undeifned uses cosmjs.